### PR TITLE
Arguments supports for start command

### PR DESCRIPTION
**Pull request by unbug**
The systemd example [https://docs.nodebb.org/configuring/running/](https://docs.nodebb.org/configuring/running/) should be 

```
[Unit]
Description=NodeBB
Documentation=https://docs.nodebb.org
After=system.slice multi-user.target

[Service]
Type=simple
User=myuser
WorkingDirectory=/path/to/nodebb
ExecStart=/path/to/nodebb/nodebb start --no-silent --no-daemon
Restart=always

[Install]
WantedBy=multi-user.target
```
Also to make it works, the start command should supports arguments.
The issue happens on `ubuntu 16`.

### CODE REVIEW

File: nodebb [hunk 1]
```diff
@@ -308,7 +308,7 @@ var commands = {
 			process.stdout.write('  "' + './nodebb restart'.yellow + '" to restart NodeBB\n\n'.reset);
 
 			// Spawn a new NodeBB process
-			cproc.fork(loaderPath, {
+			cproc.fork(loaderPath, [].concat(process.argv.slice(3)), {
```
Comment (pitaj) [inline nodebb]: Shouldn't need `[].concat`

**Comment by barisusakli** [comment]
slice(3) is correct right? first element is process.execPath second is the file being executed. The rest are the params, are you skipping the third because it is `start` ?

**Comment by unbug** [comment]
@barisusakli The start is the own command right? See my example.

**Comment by barisusakli** [comment]
Yeah thats what I mean so your code skips `start` and passes in `--no-silent --no-daemon` , looks good to me.

**Comment by unbug** [comment]
It's working great for me, check out [WebAssembly Rocks](https://www.wasmrocks.com/).

**Comment by barisusakli** [comment]
Can you send this PR against master?

**Comment by unbug** [comment]
There're conflicts, I think merge to `v1.5.x` should be the right thing to do.

**Comment by pitaj** [comment]
We don't merge directly into any version branches. Target master.

**Pull request merged**